parts/linux/cloud-init/artifacts/cse_start.sh

```diff
@@ -1,12 +1,19 @@
 CSE_STARTTIME=$(date)
+CSE_STARTTIME_FORMATTED=$(date +"%F %T.%3N")
 timeout -k5s 15m /bin/bash /opt/azure/containers/provision.sh >> /var/log/azure/cluster-provision.log 2>&1
 EXIT_CODE=$?
 systemctl --no-pager -l status kubelet >> /var/log/azure/cluster-provision-cse-output.log 2>&1
 OUTPUT=$(tail -c 3000 "/var/log/azure/cluster-provision.log")
 KERNEL_STARTTIME=$(systemctl show -p KernelTimestamp | sed -e  "s/KernelTimestamp=//g" || true)
+KERNEL_STARTTIME_FORMATTED=$(echo ${KERNEL_STARTTIME} | xargs -i date -d {} +"%F %T.%3N" )
```

Comment:
this _FORMATTED stuff is to make the returned time into something the tools are happy to parse as a date. 

Before: Fri 2022-05-06 21:26:31 UTC
After: 2022-05-06 21:26:31.000

If we just want to keep the formatted variable and remake the existing to the formatted output I'll make that change. Not sure if there's any reason behind the existing date convention so left it alone for now.

Reply:
xargs doesn't feel necessary here vs pipe alone but i'll allow it _shrugs_

Reply:
ah i was doing `| date -d - xyz` before and failed. Turns out it failed because typo. I felt fancy with an excuse to xargs. But will drop